pipelines/pulpcore/02-install.yml

```diff
@@ -1,40 +1,52 @@
 ---
-- name: Enable Postgresql 12 module if necessary
+- name: Setup git repo
   become: True
   hosts:
-   - "{{ forklift_server_name }}"
+   - "{{ forklift_name }}"
   vars_files:
-    - ../vars/install_base.yml
-    - ../vars/forklift_{{ pipeline_type }}.yml
-
+    - ../vars/forklift_pulpcore.yml
   tasks:
-  - name: Enable postgresql 12 module
-    ansible.builtin.dnf:
-      name: '@postgresql:12'
-      state: present
-    when:
-     - pipeline_version is defined
-     - pipeline_version == 'nightly' or pipeline_version is version('3.28', '>=')
-     - pipeline_os is defined
-     - pipeline_os is search("centos8-stream")
+    - name: Install podman-docker
+      package:
+        name: podman-docker
+        state: installed
 
-- name: install pulpcore
-  hosts:
-   - "{{ forklift_server_name }}"
-  become: True
-  vars_files:
-    - ../vars/install_base.yml
-    - ../vars/forklift_{{ pipeline_type }}.yml
-  environment:
-    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
-  pre_tasks:
-    - when: pipeline_version != 'nightly'
-      set_fact:
-        pulp_pkg_repo: "http://koji.katello.org/releases/yum/pulpcore-{{ pipeline_version }}/el{{ ansible_distribution_major_version }}/$basearch/"
-    - when: pipeline_version == 'nightly'
-      set_fact:
-        pulp_pkg_repo: "https://stagingyum.theforeman.org/pulpcore/{{ pipeline_version }}/el{{ ansible_distribution_major_version }}/$basearch/"
-        pulp_pkg_name_prefix: "python3.11-"
-  roles:
-    - epel_repositories
-    - pulp.pulp_installer.pulp_all_services
+    - name: Start podman
+      service:
+        name: podman
+        state: started
+
+    - name: Install git
+      package:
+        name: git
+        state: installed
+
+    - name: Enable ruby 2.7 module
+      command: dnf module enable -y ruby:2.7
+      when: ansible_distribution_major_version == "8"
+
+    - name: enable powertools
+      command: dnf config-manager --set-enabled powertools
+      when: ansible_distribution_major_version == "8"
+
+    - name: Install Ruby
+      package:
+        name:
+          - ruby
+          - ruby-devel
+          - rubygem-bundler
+          - gcc-c++
+          - make
+          - redhat-rpm-config
+          - libyaml-devel
+        state: installed
+
+    - name: Clone puppet-pulpcore
```

Comment:
We'll want to add a parameter to select the branch. In git master we may drop support for a version that still has pipelines. If we can then pin it to some stable branch it should still be good.

Reply:
You could include the role from https://github.com/theforeman/forklift/pull/1580 and then set the source repository and any number of desired remotes and branches in `pipelines/vars/forklift_pulpcore.yml`

That handles the bundle install as well if you add `install_gems: true`